packages/platform-android/src/commands/runAndroid/index.ts

```diff
@@ -314,24 +313,34 @@ function startServerInNewWindow(
 
   if (process.platform === 'darwin') {
     try {
-      return execa.sync(
+      return await execa(
         'open',
         ['-a', terminal, launchPackagerScript],
         procConfig,
       );
     } catch (error) {
-      return execa.sync('open', [launchPackagerScript], procConfig);
+      return await execa('open', [launchPackagerScript], procConfig);
     }
   }
   if (process.platform === 'linux') {
+    // Awaiting this causes the CLI to hang indefinitely, so this must execute without .sync.
     try {
-      return execa.sync(terminal, ['-e', `sh ${launchPackagerScript}`], {
+      return await execa(terminal, ['-e', 'sh', launchPackagerScript], {
```

Comment:
`return await` is quite redundant, since we're returning a promise anyway. Can we simplify it to just `return`?

Reply:
Don't think so, since we want the try/catch to catch its error. Not sure if that happens if you just `return`

Reply:
Hm, ok.